app/scripts/models/user.js

```diff
@@ -69,6 +69,14 @@ define(function (require, exports, module) {
       });
     },
 
+    /**
+     * Log the number of stored accounts
+     */
+    logNumStoredAccounts () {
+      const numAccounts = Object.keys(this._accounts()).length;
+      this._metrics.logEventOnce(`num.accounts.${numAccounts}`);
```

Comment:
I'd prefer to make transformations as late as possible for flexibility and so we can visualize data in different ways in the future.

> Yeah agreed, this event should either use the .histogram

We tried using .histogram way back in the day and had some issues, I forget what they were, but I vaguely remember .histogram being treated identically to .count items. Maybe that's been fixed, or there's an extremely high chance I'm mis-remembering. Can histograms be done with normal .count items in DD?

> or bucket the numAccounts

I understand the case of outliers, though can the same be done in later stages (DD) so we still retain flexibility?